Week-5/assignment-4/calculator.test.js

```diff
@@ -0,0 +1,16 @@
+const cal = require('./calculator');
+
+describe('Calculator test',
+()=>{
+    test('Addition of 5 & 4 should be 9',()=>{
+        expect(cal.sum(5,4)).toBe(9);
```

Comment:
[For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..

Reply:
Done ...

![Screenshot (191)](https://user-images.githubusercontent.com/81100418/173226830-1a79b0e0-7e7a-479a-b9fc-8c2bfc5bc4a6.png)